src/examples/using-with-cmake/CMakeLists.txt

```diff
@@ -40,6 +60,13 @@ find_package(axom REQUIRED
              NO_DEFAULT_PATH 
              PATHS ${AXOM_DIR}/lib/cmake)
 
+# BEGIN FIXME:
+# Create fake empty target, this stops CMake from adding -lcuda_runtime to the link line
+add_library(cuda_runtime INTERFACE)
+
+# Create fake empty target, this stops CMake from adding -lblt_hip_runtime to the link line
+add_library(blt_hip_runtime INTERFACE)
+# END FIXME
```

Comment:
@white238 -- is there a better way to do this?

Reply:
I'd be curious where these are coming from.  Axom should be properly exporting it's targets here:

https://github.com/LLNL/axom/blob/78d987f498237eca6e9252e7869a161ac6aa80ca/src/cmake/thirdparty/SetupAxomThirdParty.cmake#L282-L297

(side note: this should be replaced with BLT's macro, `blt_export_tpl_targets`.)

Reply:
I'm not sure. I spent some time with `blt_print_target_properties`, but it was still unclear to me where this being pulled in. Maybe once LLNL/blt#590 becomes available it'll be easier to pinpoint.